csfieldguide/chapters/content/en/human-computer-interaction/sections/user-experience.md

```diff
@@ -0,0 +1,559 @@
+
+{panel end}
+
+When drawing our wireframes it was important for us to think about what the user would be doing, thinking and feeling when interacting with the app.
+For example, in our MVP we’ve said that our app will give the user a simple survey about how they are feeling.
+It’s pretty likely that they are going to be tired first thing in the morning and last thing at night, so our app should reflect that.
+A larger survey with long text answers would be overwhelming to the user while they are in this state.
+Instead we’ve gone with a simple one word survey, and we’ve associated an emoji with each answer so the user has a visual cue to help them answer the question.
+
+Wireframes are an example of a *Low Fidelity Prototype*, i.e. they are simple and test broad functionality and concepts (such as the flow rather than the visual appearance of the app).
+
+### Step 8: Test
+
+**UX Method: Usability Testing with Low Fidelity Prototypes**
+
+Before implementing the wireframes, we’ll test them with target users first.
+This will give us a chance to see if the design we have proposed is actually working the way we intended it to.
+But more importantly, by testing with real users we will be able to see if we have actually designed a viable solution to the problem.
+We want to take particular note of what they think of our solution (e.g. does it look to be helpful in addressing their needs) and how it makes them feel (e.g. does it make them feel more motivated and able to improve their sleep habits?).
+
+There are many different ways to test a low fidelity prototype.
+One way is to implement the wireframe on a tool like PowerPoint or Keynote, where clicking a button goes to a different slide.
+Another common approach is called Paper Prototyping.
+This is when you model the interface using paper and act as the computer by manually changing the state in real time, i.e. move pieces of paper around to mimic the change in the interface when the user “clicks” buttons.
+Pick a method that works for you and test it out on a few different people.
+
+During the tests take note of what goes well and what doesn’t go so well.
+Are there any pain points? Are the instructions clear?
+Does the user understand how to use the app?
+Remember that negative feedback is just as useful as positive feedback.
+If users feel confused by some actions, then take that on board and adjust your design.
+You may find that you want to repeat this step more than once before proceeding.
+
+### Step 9: Create a high fidelity prototype
+
+**UX Method: Usability Testing with High Fidelity Prototypes**
+
+In steps 1 - 8 we researched our user and used this to understand the features and flow for our app.
+Now we are ready to make a High Fidelity Prototype.
+Recall that a Low Fidelity Prototype tests the flow and basic functionality of an app, whereas a High Fidelity Prototype should appear and function very similarly to the actual app that will be released.
```

Comment:
low/high fidelity prototype is inconsistently capitalised throughout the section